proxy-contract/main.mligo

```diff
@@ -0,0 +1,13 @@
+#import "errors.mligo" "Errors"
+#import "parameter.mligo" "Parameter"
+#import "storage.mligo" "Storage"
+
+type address_update = Parameter.Types.t
+type storage = Storage.Types.t 
+
+let main (address_update, storage : address_update * storage) = 
+  let { key; address; signature } = address_update in 
```

Comment:
This is not working.
There is some security issues:
- Anyone can submit an operation signed by the originator of the contract and erase with deprecated values. When you want to now the sign an operation access to `Tezos.get_sender ()` do not use extra parameters
- Any contract can add its key to the storage.

All you need to do is:
- add a `manager_address` in the storage, and originate with the address that originate the contract
- check `Tezos.get_sender () = manager_address` => update contract addresses in the storage is an OPS operation we want to do whenever a new contract is originated (so probably in CI)